### PR TITLE
NavDisplay: add Terrain on ND

### PR DESCRIPTION
**Pull request by legoboyvdlp**
See Diff

### CODE REVIEW

File: Models/Instruments/ND/canvas/loaders.nas [hunk 1]
```diff
@@ -28,7 +28,7 @@ setlistener("/nasal/canvas/loaded", func() {
 		#var code = call(compile, [code], var err=[]);
 		var code = call(func compile(code, file), [code], var err=[]);
 		if (size(err)) {
-			#print("handling error");
+			print("handling error");
```
Comment (legoboyvdlp) [inline Models/Instruments/ND/canvas/loaders.nas]: Is this still necessary?

File: Models/Instruments/ND/canvas/ND.nas [hunk 2]
```diff
@@ -417,14 +418,14 @@ for (i = 0; i < 2; i = i + 1 ) {
...
 
-setlistener("/instrumentation/efis[0]/nd/terrain-on-nd", func{
-	var terr_on_hd = getprop("instrumentation/efis[0]/nd/terrain-on-nd");
-	var alpha = 1;
-	if (terr_on_hd) {
-		alpha = 0.5;
-	}
-	nd_display.main.setColorBackground(0,0,0,alpha);
-});
+#setlistener("/instrumentation/efis[0]/nd/terrain-on-nd", func{
```
Comment (legoboyvdlp) [inline Models/Instruments/ND/canvas/ND.nas:458]: What are these commented out changes, are they still WIP?

File: Models/Instruments/ND/canvas/loaders.nas (hunk 1 above)
Comment (inuyaksa) [reply Models/Instruments/ND/canvas/loaders.nas]: Nope, I'll comment out

File: Models/Instruments/ND/canvas/ND.nas (hunk 2 above)
Comment (inuyaksa) [reply Models/Instruments/ND/canvas/ND.nas:458]: It seems useless on my implementation. 
What it does?

Comment (legoboyvdlp) [reply Models/Instruments/ND/canvas/ND.nas:458]: OK,then it can be removed entirely?

File: Models/Instruments/ND/canvas/map/TERRAIN.symbol [hunk 3]
```diff
@@ -200,6 +201,10 @@ var init = func {
 		me.request_clear = 1;
 	});
 
+	setlistener("/ECAM/warnings/timer/nav-gpws-terr-fault", func{
```
Comment (legoboyvdlp) [inline Models/Instruments/ND/canvas/map/TERRAIN.symbol]: See Systems/A320-fwc.xml, line 1480

Better to just add a condition on the ta-tcf button, I think.

**Pull request merged**